sql/core/src/main/scala/org/apache/spark/sql/execution/command/CommandUtils.scala

```diff
@@ -75,6 +77,10 @@ object CommandUtils extends Logging {
         }.sum
       }
     }
+    val partInfo = if (partitions.nonEmpty) s" with ${partitions.length} partitions" else ""
+    logInfo(s"It took ${(System.nanoTime() - startTime) / (1000 * 1000)} ms to calculate" +
```

Comment:
Because the two branches above differ in several ways, it might be cleaner to just put two different log statements in the branches above instead of this

Reply:
If I put two different logs in the branches, I would need to have two `totalSize` values in two branches and return them after the logs. Besides, the majority of two logs would still be the same (except the partition info). So the code may look redundant that way... what do you think?

Reply:
@maropu @srowen Maybe I could change back to the initial version, which prints a log with partition info in the branch for partitioned table?
```
logInfo(s"Starting to calculate sizes for ${partitions.length} partitions.")
```
In this way we keep the "partitioned table" logic only in that branch. Then the final log applies to both non-partitioned and partitioned tables.
```
logInfo(s"It took ${(System.nanoTime() - startTime) / (1000 * 1000)} ms to calculate" +
        s" the total size for table ${catalogTable.identifier}.")
```

Reply:
I don't feel strongly about it, just sounded simpler to me to keep the current structure

Reply:
@srowen @maropu I did the changes as described above, please see if it's better